docs/release_notes.md

```diff
@@ -1,11 +1,14 @@
+# Release Notes
+
 Below is a summary of the the main features/bug fixes in the most recent releases.
 
-`Note:` For .NET Core, please refer details found [here](https://github.com/Azure/aspnet-redis-providers/wiki/Configuration#using-session-state-with-aspnet-core).
-`Note:` .NET 4.0 is not supported on StackExchange.Redis.StorngName version 1.2.3 and higher; if you need .NET 4.0, please stick with 1.2.1.
+**Note:** For .NET Core, please refer details found [here](configuration.md#using-session-state-with-aspnet-core).
+
+**Note:** .NET 4.0 is not supported on      `StackExchange.Redis.StrongName` version 1.2.3 and higher; if you need .NET 4.0, please stick with 1.2.1.
```

Comment:
We're no longer using the `StackExchange.Redis.StrongName` package - I think you can remove this line entirely